yarn.lock

```diff
@@ -0,0 +1,7031 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
We don't need yarn anymore, we can use npm directly, and the file `package-lock.json` is here to ensure package versions.

Reply:
We don’t use Yarn anymore?

Reply:
With the 3.X branch, we could swith to npm yes